reference/render/icon.html

```diff
@@ -1,10 +1,160 @@
-<span class="di di-info" aria-hidden="true"></span> <code>di di-info</code>
-<hr />
-<span class="di di-warning" aria-hidden="true"></span> <code>di di-warning</code>
-<hr />
-<span class="di di-check" aria-hidden="true"></span> <code>di di-check</code>
-<hr />
-<span class="di di-cross" aria-hidden="true"></span> <code>di di-cross</code>
-<hr />
-<span class="di di-forbidden" aria-hidden="true"></span> <code>di di-forbidden</code>
-<hr />
+<li>
```

Comment:
`li` zonder `ul` eromheen?

Reply:
Ja. De `reference` files zijn 1-op-1 compilations van de bijbehorende componenten.

In dit specifieke geval is het een compile van `icon.hbs` met de _collated_ context variants uit `icon.config.js`. Hier heb ik een constructie gemaakt dat elke variant in een `<li>` is genest. In de sibling `_preview.hbs` staat `<ol>{{{ yield }}}</ol>` waar ik met wat CSS een grid van heb gemaakt.